src/Commands/StatusCommand.php

```diff
@@ -21,12 +21,29 @@ public function handle() : int
             return collect($locale)->map(function ($internalLocale) use ($poeditorTranslations) {
                 $localTranslations = app(TranslationManager::class)->getTranslations($internalLocale);
 
-                if ($poeditorTranslations === $localTranslations) {
+                $poeditorTranslations = collect($poeditorTranslations)->dot()->sort();
+                $localTranslations = collect($localTranslations)->dot()->sort();
+
+                if ($poeditorTranslations->toArray() === $localTranslations->toArray()) {
                     return true;
                 }
 
                 $this->error("The translations for '{$internalLocale}' do not match the ones on POEditor.");
 
+                $missingLocally = $poeditorTranslations->diff($localTranslations);
+                $missingOnPoeditor = $localTranslations->diff($poeditorTranslations);
+
+                $this->table(
+                    ['Translation Key', 'Locale'],
+                    $missingLocally->merge($missingOnPoeditor)
+                        ->map(function ($value, $key) use ($internalLocale) {
+                            return [
+                                $key,
+                                $internalLocale,
+                            ];
+                        })->all()
+                );
```

Comment:
```suggestion
                $outdatedLocalTranslations = $poeditorTranslations->diff($localTranslations);
                $outdatedPoeditorTranslations = $localTranslations->diff($poeditorTranslations);

                $this->table(
                    ['Translation Key'],
                    $outdatedLocalTranslations->merge($outdatedPoeditorTranslations)->keys()->map(fn ($key) => [$key])->all()
                );
```
aangezien we toch al erboven de error "The translations for '{$internalLocale}' do not match the ones on POEditor." tonen, weten we toch al de locale; dus mss wat nutteloos om een "locale" kolom ook nog toevoegen?

kunnen we in de test ook nog de table asserten

kunnen we ook nog een test toevoegen waarbij:
- de vertalingen in verkeerde volgorde staan
- waarbij er vertaling lokaal bestaat maar niet op poeditor (of bestaat dat al?)
- waarbij er vertaling niet overeenkomt met die op poeditor

Reply:
De locale kolom lijkt me ergens toch handig. Op die manier moet je niet naar boven scrollen zoekende naar de locale. (Geen idee hoeveel lijnen we zo gemiddeld gaan hebben bij deze check. Als alles goed gaat waarschijnlijk geen, maar als er veel verkeerd staat kan dat toch redelijk wat zijn.)